community/kernel/src/main/java/org/neo4j/kernel/impl/newapi/NodeLabelIndexCursor.java

```diff
@@ -109,7 +109,8 @@ public boolean next()
         }
         else
         {
-            return innerNext();
+            LabelScanValueIndexProgressor p = progressor;
```

Comment:
This change does not seem to do anything? The same code exists in `innerNext`, right?